.goreleaser.yml

```diff
@@ -189,3 +189,12 @@ blobs:
     region: us-west-2
     folder: "{{ .Env.S3FOLDER }}/archives/{{ .Version }}"
     disable: "{{ .Env.DRY_RUN }}" 
+
+dockers:
+  - ids:
+      - confluent-alpine-amd64
+    image_templates:
+      - "confluentinc/confluent-cli:{{ .Version }}"
+      - "confluentinc/confluent-cli:latest"
+    skip_push: "{{ .Env.DRY_RUN }}"
```

Comment:
Are we waiting on the goreleaser triage ticket for this line to work?

Reply:
Sure, we can wait! No rush to merge this PR.